api/candlepin-api-spec.yaml

```diff
@@ -918,6 +930,90 @@ paths:
         default:
           $ref: '#/components/responses/default'
 
+  /subscriptions:
+    get:
+      description: Retrieves a list of Subscriptions
+      tags:
+        - Subscription
+      operationId: getSubscriptions
+      security: []
+      responses:
+        200:
+          description: A list of subscriptions
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/SubscriptionDTO'
+        default:
+          $ref: '#/components/responses/default'
+
+    post:
+      description: Activates a Subscription
+      tags:
+        - Subscription
+      operationId: activateSubscription
+      security: []
+      x-java-response:
+        type: javax.ws.rs.core.Response
+      parameters:
+        - name: consumer_uuid
+          in: query
+          description: The UUID of the consumer of the subscription
+          required: true
+          schema:
+            type: string
+        - name: email
+          in: query
+          description: The email to send a notification to for the activation
+          required: true
+          schema:
+            type: string
+        - name: email_locale
+          in: query
+          description: The locale of the email to send a notification to for the activation
+          required: true
+          schema:
+            type: string
+      responses:
+        202:
+          description: A Subscription is being activated
+        400:
+          description: Invalid request such as missing email/locale or when the consumer with the given uuid was not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
+        default:
+          $ref: '#/components/responses/default'
+
+  /subscriptions/{id}:
+    delete:
+      description: Removes a Subscription
+      tags:
+        - Subscription
+      operationId: deleteSubscription
+      security: []
+      parameters:
+        - name: id
+          in: path
+          description: The ID of the subscription to remove
+          required: true
+          schema:
+            type: string
+      responses:
+        204:
+          description: Subscription was successfully deleted
+        404:
+          description: Subscription could not be found using the provided ID
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ExceptionMessage'
```

Comment:
Can we please add `example` property here.  Something like in https://github.com/candlepin/candlepin/pull/2684
it will allow YAML file to be consistent.

Reply:
Done